### ISSUE TITLE
Need more explanation in steps 3 and 4

### ISSUE THREAD

**Issue by bpohl** [1000001101000/Debian_on_Buffalo#20]
For the record, I'm working with a LS220D.

I've gotten up through running the installer, but now I'm stuck.  Actually I was stuck before the installer but checked out the link to OpenLinkstation and discovered that the password to get to the installer was 'install', something not mentioned in this project.

So, I've made it through the install and supposedly gave it a root password and an admin account, and then at the end it says to remove the install media.  Of course there is no separate install media so when I reboot it just returns to the installer.  Then, if I go to a shell it says that the drive is mounted in /target which doesn't exist, there is nothing I recognize as a disk block device in /dev, and when I exit it doesn't return to the installer but to a console log listing my failed attempts to login with the root or admin I set.

I guess the question is what do I do now to boot to the new Debian system?  If you can give me an answer, it needs to be put in the step instructions.

**Comment by lw65** [comment]
yup ,add user /password  to new user. 
another  question first install debian with installer@ ,reboot ,it need reinstall once more . after second installed .it is normal ,anybody same ?  it need remove initrd.buffalo and image.buffalo before reboot ?

**Comment by 1000001101000** [comment]
I added the username/pass to the instructions. I’ve also started playing around with the wiki feature so I can make maintaining the docs easier going forward.

The reason folks sometimes boot back into the installer is that they don’t specify sda1/md0 as /boot. at the end of the installation process new .buffalo files are generated but if /boot isn’t specified they don’t end up in the right place.

I just got done testing an update to the installer that makes it harder to make that mistake by not displaying the guided partitioning options.

**Comment by 1000001101000** [comment]
I've tried to address this more in the new wiki instructions, let me know if you see anything more I should add.

**Comment by bpohl** [comment]
Sorry for the delay in replying.  I had to set this project aside for a couple weeks.

You were exactly right.  I didn't have the files in the first partition on the disk.  Mostly it was my lack of familiarity with mdadm and Linux raid.  All I knew was there were partitions I couldn't read and I was afraid to mess with them.  By the time I figured it out I had already found an unused drive to practice on.

So, I did connect the blank drive to my Ubuntu box with a USB-to-SATA dock and formatted it with a GPT and regular primary partitions.  I could then mount the first partition and copy the installer files to it.  Once I installed the drive in the LS the install went smoothly.

Now that I've done the same thing with the larger drives I was already using I can configure the system the way I want.  For that I started a [Linkstation-Scripts](https://github.com/bpohl/Linkstation-Scripts) GitHub project to share what I come up with.  So far it is only a init script to set the disk LEDs to show disk access.

Thanks for the help.
